inc/admin/namespace.php

```diff
@@ -189,6 +190,14 @@ function settings_fields() {
 		<?php
 	}, $settings_section, 'sso_settings' );
 
+	register_setting( $settings_section, 'sso_login_name' );
+	add_settings_field( 'sso_login_name', __( 'SSO Login Link Name', 'wp-simple-saml' ), function () use ( $options ) {
+		$value = $options['sso_login_name'];
+		?>
+		<input type="text" name="sso_login_name" id="sso_login_name" value="<?php echo esc_html( $value ); ?>" placeholder="SSO Login">
```

Comment:
This should be `esc_attr` rather than `esc_html`

Reply:
Change done.